rclcpp/include/rclcpp/node_interfaces/node_parameters_interface.hpp

```diff
@@ -137,8 +137,7 @@ class NodeParametersInterface
   RCLCPP_PUBLIC
   virtual
   rcl_interfaces::msg::SetParametersResult
-  set_parameters_atomically(
-    const std::vector<rclcpp::Parameter> & parameters) = 0;
+  set_parameters_atomically(const std::vector<rclcpp::Parameter> & parameters) = 0;
```

Comment:
do we really need this change?

Reply:
No we don't.
Just a leftover from reverting one of the commits.

Reply:
Fixed in 741fb10